src/style.css

```diff
@@ -0,0 +1,138 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: "Heavy";
+  font-weight: 800;
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search-form-big {
+  width: 100%;
+  height: 70px;
+}
+
+.search-form-small {
+  width: 100%;
+  height: 42px;
+}
+
+.big-label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.small-label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.big-label:after {
+  position: absolute;
+  content: "";
+  display: block;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.small-label:after {
+  position: absolute;
+  content: "";
+  display: block;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search-input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: inherit;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+  outline: none;
+}
+
+.search-input-big {
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search-input-small {
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.search-input-big::placeholder {
+  font-family: "Avenir", sans-serif;
```

Comment:
inherit?

Reply:
it was indicated in Readme that "Remember that inputs and other interactive elements don’t inherit font styles by default." - I understand it like it is necessary to add this rule in placeholder  but  you are right it is inherited